bitaxe-sentry/docker-compose.yml

```
@@ -0,0 +1,33 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: bitaxe-sentry_web_1
+      APP_PORT: 7070
+
+  sentry:
+    image: zachprice105/bitaxe-sentry:v0.3.2@sha256:993b42d418aed7efab950d7447cbc6783f76fe110009383bf5b7a84a64cb589b
+    user: "1000:1000"
+    restart: on-failure
+    stop_grace_period: 1m
+    volumes:
+      - bitaxe_data:/var/lib/bitaxe
```

Please use volumes like this. Add a `data/sentry` folder and a `.gitkeep` file to the PR.
```suggestion
      - ${APP_DATA_DIR}/data/sentry:/var/lib/bitaxe
```